pages/parent/signup.tsx

```diff
@@ -0,0 +1,516 @@
+        postalCode: postalCode,
+        setPostalCode: setPostalCode,
+        school: school,
+        setSchool: setSchool,
+        grade: grade,
+        setGrade: setGrade,
+        // start of learning info
+        hasLearningDifficulties: hasLearningDifficulties,
+        setHasLearningDifficulties: setHasLearningDifficulties,
+        hasPhysicalDifficulties: hasPhysicalDifficulties,
+        setHasPhysicalDifficulties: setHasPhysicalDifficulties,
+        hasSensoryDifficulties: hasSensoryDifficulties,
+        setHasSensoryDifficulties: setHasSensoryDifficulties,
+        participantDifficulties: participantDifficulties,
+        setParticipantDifficulties: setParticipantDifficulties,
+        hasOtherDifficulties: hasOtherDifficulties,
+        setHasOtherDifficulties: setHasOtherDifficulties,
+        otherDifficulties: otherDifficulties,
+        setOtherDifficulties: setOtherDifficulties,
+        specialEd: specialEd,
+        setSpecialEd: setSpecialEd,
+        physiotherapy: physiotherapy,
+        setPhysiotherapy: setPhysiotherapy,
+        speechTherapy: speechTherapy,
+        setSpeechTherapy: setSpeechTherapy,
+        occupationalTherapy: occupationalTherapy,
+        setOccupationalTherapy: setOccupationalTherapy,
+        counseling: counseling,
+        setCounseling: setCounseling,
+        artTherapy: artTherapy,
+        setArtTherapy: setArtTherapy,
+        participantTherapy: participantTherapy,
+        setParticipantTherapy: setParticipantTherapy,
+        hasOtherTherapy: hasOtherTherapy,
+        setHasOtherTherapy: setHasOtherTherapy,
+        otherTherapy: otherTherapy,
+        setOtherTherapy: setOtherTherapy,
+        guardianExpectations: guardianExpectations,
+        setGuardianExpectations: setGuardianExpectations,
+        otherDifficultyDetails: otherDifficultyDetails,
+        otherTherapyDetails: otherTherapyDetails,
+        // start of emergency info
+        parentFirstName: parentFirstName,
+        setParentFirstName: setParentFirstName,
+        parentLastName: parentLastName,
+        setParentLastName: setParentLastName,
+        parentPhoneNumber: parentPhoneNumber,
+        setParentPhoneNumber: setParentPhoneNumber,
+        parentRelationship: parentRelationship,
+        setParentRelationship: setParentRelationship,
+        // start of heard about information
+        heardFromFriendsAndFam: heardFromFriendsAndFam,
+        setHeardFromFriendsAndFam: setHeardFromFriendsAndFam,
+        heardFromFlyers: heardFromFlyers,
+        setHeardFromFlyers: setHeardFromFlyers,
+        heardFromEmail: heardFromEmail,
+        setHeardFromEmail: setHeardFromEmail,
+        heardFromSocialMedia: heardFromSocialMedia,
+        setHeardFromSocialMedia: setHeardFromSocialMedia,
+        heardFromOther: heardFromOther,
+        setHeardFromOther: setHeardFromOther,
+        heardFromOptions: heardFromOptions,
+        setHeardFromOptions: setHeardFromOptions,
+    };
```

Comment:
I feel like we could come back to this in another ticket and use some state management system like unstated-next to abstract this long piece of stub away. Along with all the related method calls. No need to change anything for this PR.